style.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital@0;1&display=swap');
+
+html {
+  box-sizing: border-box;
+  font-size: 10px;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  font-size: 1.5rem;
+}
+
+.headline-wrapper {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  border-bottom-left-radius: 110px;
+  background-image: url(file:///C:/Users/VS47~1/AppData/Local/Temp/header_section_background.png);
+  background-size: 60%;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+a {
+  color: #6070ff;
+  text-decoration: none;
+}
```

Comment:
- [ ] Please check colors on Figma and use exactly same colors, eg. `<p>` tag has `#344563` color while in your case it is `#6070ff` . Here is how you can check properties on Figma: 

![chrome-capture (9)](https://user-images.githubusercontent.com/77038610/143391146-0490ff46-ba06-4326-a507-2771deea8959.gif)


Click on elements and check the right menu, there you will see all the specifications 💯 
- [ ] Please fix the colors of other elements accordingly 👍🏻